src/modules/home/types/accommodation-amenity.type.ts

```diff
@@ -0,0 +1,21 @@
+export interface AccommodationAmenity {
+  id: string;
+  hasWifi: boolean;
+  hasParking: boolean;
+  hasSwimmingPool: boolean;
```

Comment:
Is here everything couldn't be nullable?

Reply:
> Is here everything couldn't be nullable?

Fixed

Reply:
Don't see that it is fixed

Reply:
> Don't see that it is fixed

Sorry, I thought I wrote that "fixed" on address type. Everything in amenity type is non-nullable, that's why they were left as only boolean.